core/src/main/java/com/datastax/oss/driver/internal/core/metrics/DropwizardMetricsFactory.java

```diff
@@ -73,6 +96,21 @@ public DropwizardMetricsFactory(InternalDriverContext context) {
     }
   }
 
+  private Duration getAndValidateEvictionTime(DriverExecutionProfile config) {
+    Duration evictionTime = config.getDuration(DefaultDriverOption.METRICS_NODE_EXPIRE_AFTER);
+
+    if (evictionTime.compareTo(LOWEST_ACCEPTABLE_EVICTION_TIME) < 0) {
+      LOG.warn(
+          "The {} setting was provided with too low value: {}. Consider increasing it to at least {} hour. "
+              + "Having lower values may cause your node-level metrics to keep disappearing and reappearing.",
+          DefaultDriverOption.METRICS_NODE_EXPIRE_AFTER,
+          evictionTime,
+          LOWEST_ACCEPTABLE_EVICTION_TIME.toHours());
```

Comment:
```suggestion
      LOG.warn(
          "[{}] Value too low for {}: {}. Forcing to {} instead.",
          logPrefix,
          DefaultDriverOption.METRICS_NODE_EXPIRE_AFTER.getPath(),
          evictionTime,
          LOWEST_ACCEPTABLE_EVICTION_TIME);
```
Log prefix + using the option path instead of the programmatic name + shortening the error message (I don't think it's necessary to explain the consequences) + indicating that we force the value, it's not just a suggestion + keep the raw duration, per my other comment to maybe use less than one hour.

Note that we have a pending ticket to improve the formatting of durations: [JAVA-2845](https://datastax-oss.atlassian.net/browse/JAVA-2845).